### ISSUE TITLE
Fails to load Perl module Text::CSV

### ISSUE THREAD

**Issue by tbrowder** [niner/Inline-Perl5#156]
Trying to use the Perl module Text::CSV:

~~~
$ cat 'text-csv.raku
#!/usr/bin/env raku
use Text::CSV:from<Perl5>;
$ chmod +x text-csv.raku
~~~

Execute the program:

~~~
./text-csv.raku
===SORRY!=== Error while compiling /usr/local/git-repos/csv2txf/raku/./text-csv.raku
Could not build C3 linearization: ambiguous hierarchy
at /usr/local/git-repos/csv2txf/raku/./text-csv.raku:2
~~~

Environment:

Debian 10 (Buster)

~~~
$ raku -v
Welcome to 𝐑𝐚𝐤𝐮𝐝𝐨™ v2020.10.
Implementing the 𝐑𝐚𝐤𝐮™ programming language v6.d.
Built on MoarVM version 2020.10.
~~~

**Comment by tbrowder** [comment]
Ping?

**Comment by niner** [comment]
Apparently, Text::CSV's inheritance graph doesn't lend itself to C3 linearization:
```
> perl -E 'use mro; use Text::CSV; say mro::get_linear_isa("Text::CSV", "c3")'
Inconsistent hierarchy during C3 merge of class 'Text::CSV':
        current merge results [
                Text::CSV,
        ]
        merging failed on 'Exporter' at -e line 1.
```

Right now Inline::Perl5 uses Raku's Metamodel::C3MRO unconditionally. Seems like it needs to be smarter and use DFS where appropriate. I do wonder how the 2 can be mixed though.

**Comment by tbrowder** [comment]
I just appreciate all you do, so if it can't feasibly be done, so be it!

**Comment by niner** [comment]
Fixed in commit 33439d34b37fd081ef1f64f0894f07f91cd5911b
Thanks for the report! Always learning something new about Perl...
